scripts/getchoco.ps1

```diff
@@ -32,6 +32,7 @@
 $url = ''
 
 try {
+    $env:chocolateyVersion = '1.4.0'
     $chocolateyVersion = $env:chocolateyVersion
```

Comment:
You need a log message to explain this! Not a comment!
Write-Host("$(Log-Date) On Azure, choco 2.0 requires .Net 4.8. Firstly this requires the caller to action a reboot. Then when executing choco from a remote session it fails to locate .Net 4.8. If you RDP in, choco works fine. Note that AWS does not have this issue so there is a setting somewhere but this could not be located.")